src/gui/LayoutMapper.cpp

```diff
@@ -1,42 +1,22 @@
 #include "gui/LayoutMapper.h"
 
 
-/**
- * compileLayout - assemble bitflags to create basic layouts
- */
-LayoutType compileLayout(bool isVertical, bool isRightToLeft, bool isBottomToTop)
-{
-	int type = isVertical ? LayoutBitFlags::Vertically : 0;
-	type |= isRightToLeft ? LayoutBitFlags::RightToLeft : 0;
-	type |= isBottomToTop ? LayoutBitFlags::BottomToTop : 0;
-
-	return (LayoutType) type;
-}
-
-LayoutMapper::LayoutMapper(int pages, int numRows, int numCols, bool useRows, LayoutType type, bool isPaired,
-		int firstPageOffset)
-{
-
+LayoutMapper::LayoutMapper(){
+	
 	XOJ_INIT_TYPE(LayoutMapper);
-
-	layoutMapperInit(pages, numRows, numCols, useRows, type, isPaired, firstPageOffset);
-
+	
+	this->actualPages = 0;
```

Comment:
We use a really recent C++ Version, which allow to initialize vars direct in the header file, really makes sense I think, even if it's not yet that common in C++.

This is not a problem or so, only a tip ;-)

Reply:
Thanks,  Keep 'em coming!